extContractsGroup.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "contivModel",
+	"objects": [
+		{
+			"name": "extContractsGroup",
+			"type": "object",
+			"key": [ "contractsGroupName" ],
```

Comment:
Contracts typically have tenant scope -- the tenant common is the only exception. May be better to keep the scope consistent.